### DISCUSSION TITLE
Osu!catch dashing with mouse actually shows cursor

### DISCUSSION THREAD

**Discussion by Aimiks** [ppy/osu#21080] [Q&A] [answered]
The cursor appear and a "click" sound effect is played when using MB1 to perform dash in osu!catch.
This is distracting for players using MB1 or MB2 button to dash.

I've looked into the settings but it doesn't seem to have any options regarding this.
Is there any ?

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21088
